changelog.json

```
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.6.3": [
+      "[Improved] Upgrade embedded Git to v2.29.2 - #11369",
+      "[Improved] Enable spellcheck on commit summary and description - #1597",
+      "[Improved] Update app icon for macOS Big Sur - #10855",
```

I think this is all macOS, right?
```suggestion
      "[Improved] Update app icon for macOS - #10855",
```